src/renderer/components/notifications/notifications.tsx

```diff
@@ -37,20 +37,27 @@ export class Notifications extends React.Component {
   static ok(message: NotificationMessage) {
     notificationsStore.add({
       message,
-      timeout: 2500,
+      timeout: 2_500,
```

Comment:
There is an ESLint plugin that adds a `numeric-separators-style` rule that could be useful, I think
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/numeric-separators-style.md

Reply:
I think that should be a separate PR.